themes/scss/map/cartodb-map-light.scss

```diff
@@ -309,31 +309,31 @@ only screen and (                min-resolution: 2dppx) {
     background-position:-94px 10px!important;
   }
   div.cartodb-header div.social a.facebook {
-    background:url('../img/other@2x.png') no-repeat 0 0!Important;
-    background-size: 113px 34px!Important;
+    background:url('../img/other@2x.png') no-repeat 0 0!important;
+    background-size: 113px 34px!important;
```

Comment:
! should be preceeded by a space, and should not be followed by a space<br>!important should not be used